python/activator/activator.py

```diff
@@ -155,13 +214,8 @@ def next_visit_handler() -> Tuple[str, int]:
     status : `int`
         The HTTP response status code to return to the client.
     """
-    if request.args.get("token", "") != verification_token:
-        return "Invalid request", 400
-    subscription = subscriber.create_subscription(
-        topic=topic_path,
-        ack_deadline_seconds=60,
-    )
-    _log.debug(f"Created subscription '{subscription.name}'")
+    consumer.subscribe([bucket_topic])
```

Comment:
Because there's no explicit redelivery on unprocessed messages like Pub/Sub had, there's no need to add subscriptions dynamically, I think.  Can't we subscribe once and reuse for each visit?